packages/now-cli/test/dev/integration.js

```diff
@@ -573,7 +573,11 @@ test('[vercel dev] validate mixed routes and rewrites', async t => {
   const output = await exec(directory);
 
   t.is(output.exitCode, 1, formatOutput(output));
-  t.regex(output.stderr, /Cannot define both `routes` and `rewrites`/m);
+  t.regex(
+    output.stderr,
+    /If `rewrites`, `redirects`, `headers`, `cleanUrls` or `trailingSlash` is used, `routes` cannot be present./m
```

Comment:
I realize this isn't the place to fix it, but this error message has some grammar errors:

```suggestion
    /If `rewrites`, `redirects`, `headers`, `cleanUrls` or `trailingSlash` are used, then `routes` cannot be present./m
```